src/main/resources/static/swagger.yaml

```diff
@@ -338,6 +365,22 @@ components:
         - HIType4
         - HIType5
       example: "DiagnosticReport"
+    HIPIds:
+      type: string
+      example: "100001"
+    PatientConsentRequest:
+      type: object
+      properties:
+        hipIds:
+          type: array
+          items:
+            $ref: '#/components/schemas/HIPIds'
+    PatientConsentResponse:
+      type: object
+      properties:
+        hipId:
```

Comment:
the property will not be "hipId". I think you need to describe as

     type: object
          additionalProperties:
            type: string
          example:
            "100001": req-id-for-hip-100001
            "100002": req-id-for-hip-100002